src/actions/upload.ts

```diff
@@ -1,12 +1,15 @@
 import {runPclCommand} from '../pcl';
+import semver from 'semver';
 
 export async function uploadToPayaraCloud(
     pclExecutable: string,
     subscriptionName: string | null,
     namespace: string,
     appName: string | null,
     warFile: string,
-    isDeploy: boolean
+    isDeploy: boolean,
+    endpointUrl: string | null,
+    pclVersion: string
```

Comment:
`qubeVersion`, really. Or is you really want to keep backwards compatibility then add `qubeVersion` in addition for `pclEndpoint`